python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_condition.py

```diff
@@ -783,7 +804,7 @@ def condition(self) -> AutomationCondition:
 
     @property
     def condition_unique_id(self) -> str:
-        return self._context.condition_unique_id
+        return self._context.condition_unique_ids[0]
```

Comment:
do we need to worry at all about this property now being a footgun?

Reply:
I think we need to choose a single unique identifier at some point for this to work, so this feels ok.